README.md

```diff
@@ -83,11 +95,7 @@ API documentation can be found at [https://readthedocs.org/projects/text-extensi
   cover complex end-to-end NLP use cases (work in progress).
 
 
-## Instructions to run a demo notebook
-1. Check out a copy of this repository
-1. (optional) Use the script `env.sh` to set up an Anaconda environment for running the code in this repository.
-1. Type `jupyter lab` from the root of your local source tree to start a [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/) environment.
-1. Navigate to the `notebooks` directory and choose any of the notebooks there
+## Instructions to run a demo notebook locally
```

Comment:
This is covered above, did you mean to remove this line?

Reply:
Oops!  Fixed now.